src/services/notificationHandler.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+const NotificationHandler = require('../lib/notificationHandlerWebsocket')
```

Comment:
Everything is event driven so there is no worker anymore. Worker gives the notion that a background process is doing something, which is not the case anymore. A handler just does the right thing with your notifications.